sdk/keyvault/keyvault-common/package.json

```diff
@@ -32,7 +32,7 @@
     "integration-test:node:no-timeout": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json src --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint src --ext .ts -f html -o search-lintReport.html || exit 0",
+    "lint": "eslint package.json src --ext .ts -f html -o keyvault-common-lintReport.html || exit 0",
     "lint:terminal": "eslint src --ext .ts",
```

Comment:
```suggestion
```

Reply:
but `lint:terminal` can be useful here because `lint` creates an html report and `lint:terminal` more conveniently shows the report in the terminal.

Reply:
yes, but we havent really standardized on this. Not all packages have the `lint:terminal` script. I am ok with people having to remove the `-f` and `-o` flags to see the lint output in the terminal if they really want to rather than maintain yet another npm script in the package.json file

Reply:
I see, I opened this PR: https://github.com/Azure/azure-sdk-for-js/pull/11530.